### DISCUSSION TITLE
FIlecoin Interoperability with FVM

### DISCUSSION THREAD

**Discussion by BlocksOnAChain** [filecoin-project/community#568] [General]
We will use this discussion section to talk about potential ideas for Filecoin Interoperability, which is now made possible with the FVM project. 
As we know, Interoperability technologies allow different blockchains to interact with each other directly or via some intermediaries.

So let's talk about bridges, accessibility improvements we can have, and security management for the Filecoin network...

**Comment by wheresaddie** [comment]
A low hanging fruit might be to consider developing a sort of light client w state verification -  it could be some sort of zkp that exports FVM state to other chains - so the client could share for example, recent transactions signed by the network. This could in theory circumvent an intermediary need (like bridges) since it would come from the chain itself-  using a litght client means it the state could be easily verified inside apps on other blockchains, so we could provide any dapp in other ecosystems with information. This would solve potentially a few issues of accessibility, security and interop in one go
